packages/components/src/truncate/hook.tsx

```diff
@@ -45,14 +46,14 @@ export default function useTruncate( props ) {
 	const shouldTruncate = ellipsizeMode === TRUNCATE_TYPE.auto;
 
 	const classes = useMemo( () => {
-		const sx = {};
-
-		sx.numberOfLines = css`
-			-webkit-box-orient: vertical;
-			-webkit-line-clamp: ${ numberOfLines };
-			display: -webkit-box;
-			overflow: hidden;
-		`;
+		const sx = {
+			numberOfLines: css`
+				-webkit-box-orient: vertical;
+				-webkit-line-clamp: ${ numberOfLines };
+				display: -webkit-box;
+				overflow: hidden;
+			`,
+		};
```

Comment:
Not sure the original reason. By adding it directly into the object we don't have to add types to the object since that will be done automatically.

No strong options on this so can be changed back.

Reply:
> I don't think we need to overthink it though

Absolutely! I just asked because I've seen this pattern a few times already and wasn't aware of the reasons behind — totally cool for me to simplify code / rename variables if necessary.

> Also I'm still not that experienced with CSS-in-JS architectures, so I don't know why certain styles are written in hooks.ts and not styles.ts 🙃

I also wondered why, and I don't think I have a good answer to it yet. I'd be in favour of picking one approach and just sticking with it